### ISSUE TITLE
[BUG] Unable to delete notes in 2.00 beta rc1

### ISSUE THREAD

**Issue by muecke6** [paperless-ngx/paperless-ngx#4597]
### Description

Error delete notes in 2.00 beta rc1

### Steps to reproduce

Existing or newly created notes cannot be deleted
Newly created note overwrites existing old note

### Webserver logs

```bash
{"headers":{"normalizedNames":{},"lazyUpdate":null},"status":404,"statusText":"Not Found","url":"http://192.168.0.110:8777/api/documents/1812/notes/?id=421","ok":false,"name":"HttpErrorResponse","message":"Http failure response for http://192.168.0.110:8777/api/documents/1812/notes/?id=421: 404 Not Found","error":{"detail":"Nicht gefunden."}}
```

### Paperless-ngx version

2.00 beta rc1

### Host OS

Synology

### Installation method

Docker - official image

### Browser

Firefox

**Comment by stumpylog** [comment]
That's not actually a server log.  Please provide those.

**Comment by shamoon** [comment]
I can reproduce this, I'll sort it out. Thanks for the report

**Comment by stumpylog** [comment]
Looks like maybe a frontend thing?  The backend (via admin) removes the note, but it hangs around in the UI

**Comment by shamoon** [comment]
It's the backend, the OPTIONS call is somehow triggering the full DELETE so the second time when it actually runs DELETE it fails. I think from custom fields changes. Im trying to sort it out

![5f08851f-62fd-4402-90a1-e1be815e912b](https://github.com/paperless-ngx/paperless-ngx/assets/4887959/386e56e0-39ac-4dd7-b306-f420e5bcdc9f)
![8e73c827-25d5-4408-bd7c-a527aa29be8c](https://github.com/paperless-ngx/paperless-ngx/assets/4887959/ca5c98fe-d331-45cc-a555-af4ce3587604)
![a753358b-f345-4910-8da1-d80e00d411ca](https://github.com/paperless-ngx/paperless-ngx/assets/4887959/426e467b-9d4e-4647-9c12-a721fef9e7f5)

**Comment by stumpylog** [comment]
Maybe something [drf-writable-nested](https://github.com/beda-software/drf-writable-nested) is doing?

**Comment by shamoon** [comment]
I'm a bit stumped (ha) here, it turns out this actually is also happening in our current version it just doesnt cause an error (screenshot is from `main`):

<img width="614" alt="Screenshot 2023-11-15 at 9 50 57 AM" src="https://github.com/paperless-ngx/paperless-ngx/assets/4887959/f9e28b21-5f71-4f97-b8b2-693f45be7e81">

But I cannot for the life of me track down:

- how to detect if the request is `OPTIONS` (and thus not perform the delete). This seems like the 'proper' fix, really I wouldn't expect that we'd have to manually do this
  - `request.method` still shows `DELETE` (obviously). Looking around I didnt find a solution that correctly reveals the true method, it seems somehow like this info is lost
- why the `DELETE` (after `OPTIONS`) now throws a 404. Of course I think the real problem is point above

@stumpylog if you or anyone else has insight would be awesome. In the meantime I'll keep digging around...

**Comment by stumpylog** [comment]
I wish I had insights into DRF.  It's almost as obtuse to me as Whoosh is.  I'll try to look when I have time as well.  Or at least be a sounding board

**Comment by shamoon** [comment]
Sorry for the wild goose chase, I think the fix is incredibly simple. Hopefully no one else wasted too much time.

**Comment by shamoon** [comment]
Closed in branch by #4602
